### DISCUSSION TITLE
Perdita connettività frequente dopo aggiornamento 2022.9.7

### DISCUSSION THREAD

**Discussion by nikpait** [indomus/forum#1954] [HUB: Home Assistant]
Ciao a tutti,
da diversi giorni sono dietro a un problema dal quale non riesco ad uscire.
Ho aggiornato il mio HA (Home Assistant Supervisor) dalla versione 2022.9.6 alla versione 2022.9.7.  Da quando ho effettuato l'aggiornamento capita che ogni tot (ogni 30 minuti o ogni ora) non c'è un tempo standard, qando cerco di collegarmi sul portale https://xxxxx.duckdns.org:8123/ ricevo il messaggio impossibile collegarsi, dall' IP invece funzione tutto correttamente. L' HA è installato su un Rasperry già da diversi anni e non ho mai avuto problemi. Se lacio tutto così però dopo un pò riesco a ricollegarmi nuovamente.
Ho provato a dare un'occhiata ai log. 
Premetto che il certificato scade i primi di Novembre, ma ogni tanto vedo l'errore sotto:

Error fetching xxx.duckdns.org:8123 data: Cannot resolve hostname: xxx.duckdns.org
09:34:44 – (ERRORE) Scadenza certificato

Potete darmi qualche consiglio su cosa verificare?

Grazie

**Comment by snakuzzo** [comment]
Non c'è alcun consiglio da dare...
prova a vedere come viene risolto il tuo dominio con una lookup. Se non sai i comandi vai su https://www.nslookup.io/ e vedi se corrisponde al tuo IP pubblico. Magari ci sono problemi lato DuckDNS (so che in questi giorni ce ne sono stati)

**Reply by nikpait** [reply]
al momento ho risolto tornando alla versione 2022.9.6  e sembra funzionare tutto correttamente.
Quindi penso sia un problema legato alla versione.
Grazie

**Reply by snakuzzo** [reply]
Non mi pare proprio ci siano problemi di questo tipo...e mi sembra stranissimo che ci sia un problema così specifico su una versione specifica relativa poi ad un servizio esterno che non c'entra un fico secco con HA...a meno che tu non stia usando HA OS e qualche addon.
Posso assicurarti però che io ho anche un'installazione di HA OS con DuckDNS aggiornata all'ultima versione e non ho mai avuto alcun problema

**Reply by nikpait** [reply]
Potrebbe trattarsi dell'aggiornamento non andato a buon fine?

**Reply by snakuzzo** [reply]
no

**Reply by nikpait** [reply]
boh, allora è strano. L'unica cosa certa è che facendo il downgrade funziona correttamente

**Reply by snakuzzo** [reply]
Ripeto...in base a quello che hai detto, il tuo problema non è legato alla versione di core
